src/Type.php

```diff
@@ -276,6 +278,22 @@ public function exists($conditions)
      */
     public function save(EntityInterface $entity, $options = [])
     {
+        $type = $this->connection()->getIndex()->getType($this->name());
+        $id = $entity->id ?: null;
+
+        $data = $entity->toArray();
+        unset($data[$id]);
+        $doc = new ElasticaDocument($id, $data);
+        $doc->setAutoPopulate(true);
+
+        $result = $type->addDocument($doc);
```

Comment:
There aren't any documented, but that doesn't mean a method somewhere lower down doesn't throw exceptions.